src/pages/dmt5/_dhero.js

```diff
@@ -140,16 +148,25 @@ const InformationWrapper = styled(Flex)`
     z-index: 1;
 
     @media (max-width: 1376px) {
-        max-width: 44rem;
+        max-width: 52rem;
     }
-    @media ${device.tabletL} {
+    @media (max-width: 1270px) {
+        max-width: 46rem;
+    }
+    @media ${device.laptopM} {
         max-width: 40rem;
     }
+    @media ${device.laptop} {
+        max-width: 38rem;
+    }
     @media ${device.tablet} {
         top: 280px;
-        max-width: 328px;
+        max-width: 450px;
```

Comment:
Can we use `rem` to make it consistent with the [ones above](https://github.com/binary-com/deriv-com/pull/1274/files#diff-0d2d1b4cbc93ddfff0bfb794a3b26553242e732ef1549d80f618d30364215575R150-R161)